modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -45,28 +45,13 @@ struct GAPI_EXPORTS_W_SIMPLE OpenVINO {
 
     /** @brief Class constructor.
 
-    Constructs OpenVINO parameters based on device information.
+    Constructs OpenVINO parameters based on device type information.
 
-    @param device Target device to use.
+    @param dev_type Target device type to use. ("CPU_FP32", "GPU_FP16", etc)
```

Comment:
Here comes the question.. Should we expose it as-is or build the right facade atop of it?

Reply:
I think there is no right facade so far. Only `device_type` is used correctly by `OV EP`